volumes/frus1909.xml

```diff
@@ -34403,7 +34403,7 @@
                             1909.</note></head>
                         <opener><dateline rendition="#right"><placeName><hi rend="smallcaps"
                                         >Imperial Japanese Embassy</hi>,<lb/><hi rend="italic"
-                                        >Washington</hi></placeName> [, <date when="1909-07-17"
+                                        >Washington</hi></placeName>[, <date when="1909-07-17"
```

Comment:
Please confirm the spacing here (if the website can't open the link, it's line 34406 of frus1909).

Reply:
Correct.  See above.